developer/src/kmc-keyboard-info/test/fixtures/khmer_angkor/build/khmer_angkor.keyboard_info

```diff
@@ -15,7 +15,13 @@
         ]
       },
       "examples": [{
-        "keys": "x j m E r",
+        "keys": [
+          { "key": "x" },
+          { "key": "j" },
+          { "key": "m" },
+          { "key": "E" },
+          { "key": "r" }
+        ],
```

Comment:
So, this was the main point of this PR, right?

It would be nice to have a test fixture that tests the modifiers part of `packageKeysExamplesToKeyboardInfo`; we're only testing the raw 'key' part of it with this specific test.

Reply:
Made this into a future issue: #9621.